data/index.html

```diff
@@ -5,133 +5,125 @@
-/*
-  table {
-    border-collapse: collapse;
-  }
-  table, th, td {
-    border: 1px solid black;
-  }
-  #notice {
-    font-style: italic;
-    margin: 5px;
-    padding: 5px 10px;
-    background-color: #EEEEEE;
-  }
-  #stats {
-    font-style: italic;
-    font-size: small;
-    margin-bottom: 20px;
-  }
-  #measurements h4 {
-    margin: 10px 5px 3px 0px;
-  }
-*/
-</style>
+  <title>Firefox data explorer - Find probes</title>
 
-<script type="text/javascript"
-        src="jquery-3.1.1.min.js">
-</script>
-<script type="text/javascript" src="explore.js"></script>
-
-<!-- Latest compiled and minified CSS -->
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-
-<!-- Optional theme -->
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
-
-<!-- Latest compiled and minified JavaScript -->
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+  <!-- CSS  -->
+  <!-- CSS:Framework BS 4 alpha -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+  <!-- CSS:Framework:Theme United -->
+  <link rel="stylesheet" href="bootstrap.min.css">
+  <!-- CSS:LoadingBar -->
+  <link rel="stylesheet" href="explorer.css">
 
 </head>
 
 <body>
 
-<a id="github-banner"
-   href="https://github.com/georgf/fx-data-explorer">
-   <img src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png" />
-</a>
-
-<nav class="navbar navbar-expand-md navbar-inverse bg-inverse mb-4">
-  <a class="navbar-brand" href="#">Probe data</a>
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
-    <span class="navbar-toggler-icon"></span>
-  </button>
-  <div class="collapse navbar-collapse" id="navbarCollapse">
-    <ul class="navbar-nav mr-auto">
-      <li class="nav-item active">
-        <a class="nav-link" href="#probes">Find probes <span class="sr-only">(current)</span></a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="#stats">Stats</a>
-      </li>
-    </ul>
+  <!-- GitHub:Fork -->
+  <!-- <a id="github-banner"
```

Comment:
Instead of commenting out, lets just remove this if it doesn't fit the style.
We can open a follow-up issue to figure out where to link to the GH repo.

Reply:
Removed.